### DISCUSSION TITLE
Blank screen on Samsung TV

### DISCUSSION THREAD

**Discussion by dyter69** [bpatrik/pigallery2#861] [Q&A]
I run pigallery 2.0 on docker container synology DSM7.2 last version

pigallery is not showing on TizenOS from WebOS of Samsung TV, only pigallery logo is showing

**Comment by bpatrik** [comment]
old browsers cannot render the app. (IE 11 support was dropped by angular. i can't do nothing here)

**Comment by dyter69** [comment]
Jellyfin work fine on samsung web browser with H265 4K ! 
It's a real shame because it would be great to be able to watch my photos on my TV

**Comment by bpatrik** [comment]
Its using angular as a frontend framework. Angular dropped old IE (like IE
v9 or so) support a few years ago. I know that killed the app on older TVs.

I can't do anything here. Use screen mirroring from your phone.

**Comment by dyter69** [comment]
It's a shame because it was the best use I could have had
